design-proposals/network-binding-plugin/network-binding-plugin.md

```diff
@@ -224,6 +224,56 @@ pod.
     - Mount points
     - Downward API
 
+##### Compute Container Resource Overhead
+
+For some plugins, an additional resource consumption can be expected from the virt-launcher pod compute container.
+For example, there could be need to execute an additional binary in the compute container.
```

Comment:
Could you please explain what binary can be executed and how it gets there? 
How can an external plugin rely on the fact that a specific binary is part of the virt-launcher image?

Reply:
This example was derived from the `passt` use case, on which the `passt` binary is shipped as part of the virt-launcher `compute` image.

Another example could be that the plugin configures the domain in a way that causes the virt-stack to consume additional compute resources that KubeVirt cannot account for.